src/wallet/rpc/spend.cpp

```diff
@@ -799,6 +808,8 @@ RPCHelpMan fundrawtransaction()
                                     },
                                 },
                              },
+                            {"max_tx_weight", RPCArg::Type::NUM, RPCArg::Default{MAX_STANDARD_TX_WEIGHT}, "The maximum acceptable transaction weight.\n"
+                                                          "Transaction selection will fail if this can not be satisfied."},
```

Comment:
I think you meant either "Transaction building" or "Coin selection" here:

```suggestion
                                                          "Transaction building will fail if this can not be satisfied."},
```

```suggestion
                                                          "Coin selection will fail if this can not be satisfied."},
```